src/Storefront/Theme/ThemeMergedConfigBuilder.php

```diff
@@ -110,7 +110,18 @@ public function getPlainThemeConfiguration(string $themeId, Context $context): a
             }
         }
 
-        $themeConfig['themeTechnicalName'] = $theme->getTechnicalName();
+        // Check if the theme is a database copy of a physical theme.
+        // If so, use the technical name of the parent theme.
+        if ($theme->getTechnicalName() === null && $theme->getParentThemeId() !== null) {
+            $parentTheme = $themes->filter(fn (ThemeEntity $themeEntry) => $themeEntry->getId() === $theme->getParentThemeId())->first();
```

Comment:
`\Shopware\Core\Framework\DataAbstractionLayer\EntityCollection::filterByProperty` could also be used

Reply:
maybe a bit easier to read :man_shrugging:

Reply:
but as I already approved, you can of course ignore this :blush: